toml/_parser.ts

```diff
@@ -2,6 +2,7 @@
 // This module is browser compatible.
 
 import { deepMerge } from "@std/collections/deep-merge";
+import { isLeap } from "@std/datetime";
```

Comment:
`@std/toml` cannot depend on `@std/datetime` as `toml` is stable and `datetime` is unstable. This causes lint CI failure